courses/free-courses-en.md

```diff
@@ -125,6 +125,7 @@
 * [IIT Bombay Foundation of Data Structures (CS213.1x)](https://courses.edx.org/courses/course-v1:IITBombayX+CS213.1x+1T2017/course/)
 * [Intro to Data Structures and Algorithms](https://www.udacity.com/course/data-structures-and-algorithms-in-python--ud513) - Brynn Claypoole, Horatio Thomas (Udacity)
 * [JavaScript Algorithms and Data Structures](https://www.youtube.com/playlist?list=PLC3y8-rFHvwjPxNAKvZpdnsr41E0fCMMP) - Codevolution
+* [Linked Lists Course](https://youtube.com/playlist?list=PLDzeHZWIZsTr54_TH_NK4ibFojS4mmQA6) - LoveBabbar
```

Comment:
Don't invent titles! Use the title given by the resource.
Look at other items in the list for Babbar's courses

Reply:
Done. Also the other items by Babbar are not the same